src/jttt/Core/AlphaBetaStrategy.java

```diff
@@ -0,0 +1,112 @@
+            return calculateResult(currentBoard, depth);
+        }
+
+        for (Integer move : currentBoard.remainingPositions()) {
+            Board currentStateOfBoard = currentBoard.newBoardWithNewMove(move, currentCounter);
+            Integer score = aBMinimax(  depth - 1,
+                                        currentCounter.opponentCounter(),
+                                        alpha,
+                                        beta,
+                                        currentStateOfBoard).get(SCORE_KEY);
+            if (newBestScore(currentCounter, bestScore, score)) {
+                bestScore = score;
+                bestMove = move;
+            }
+            alpha = newAlpha(alpha, score, currentCounter);
+            beta = newBeta(beta, score, currentCounter);
+            if (beta <= alpha) {
+                break;
+            }
+        }
+        return createResultMap(bestScore, bestMove);
+    }
+
+    private int newAlpha(int alpha, Integer score, Counter currentCounter) {
+        if (isComputerCounter(currentCounter)) {
+            return max(alpha, score);
+        }
+        return alpha;
+    }
+
+    private int newBeta(int beta, Integer score, Counter currentCounter) {
+        if (!isComputerCounter(currentCounter)) {
+            return min(beta, score);
+        }
+        return beta;
+    }
+
+    private int indexToDisplayPosition(HashMap<String, Integer> result) {
+        return result.get(MOVE_KEY) + DISPLAY_POSITION_OFFSET;
+    }
+
+    private boolean newBestScore(Counter currentCounter, Integer bestScore, Integer score) {
+        return isComputerCounter(currentCounter) && score > bestScore ||
+                !isComputerCounter(currentCounter) && score < bestScore;
+    }
+
+    private boolean isComputerCounter(Counter currentCounter) {
+        return this.counter == currentCounter;
+    }
+
+    private boolean noMoreMovesAvailable(int depth, Board currentBoard) {
+        return currentBoard.isGameOver() || depth == 0;
+    }
+
+    private Integer setInitialBestScore(Counter currentCounter) {
+        return currentCounter == this.counter ? -INITIAL_SCORE : INITIAL_SCORE;
+    }
+
+    private HashMap<String, Integer> calculateResult(Board currentBoard, int depth) {
+        double score = INITIAL_SCORE / (currentBoard.boardSize() - currentBoard.numberOfOpenPositions());
+        if (currentBoard.findWinner().equals(Counter.EMPTY)) {
+            return createResultMap(SCORE_FOR_DRAW, -1);
+        } else if (currentBoard.findWinner().equals(this.counter)) {
+            score = varyScoreUsingDepth(depth, score);
+        } else {
+            score = varyScoreUsingDepth(-depth, -score);
+        }
+        return createResultMap(score, -1);
+    }
+
+    private double varyScoreUsingDepth(int depth, double score) {
+        return score + depth;
+    }
+
+    private HashMap<String, Integer> createResultMap(double score, int move) {
```

Comment:
This map feels like a little class that wants to be created.
It may help in providing a nice home for some of the constants defined at the top of the class.
I can imagine something along the line of `Result r = new Result(move, score)` and Result draw = new Draw(move)`